rero_ils/permissions.py

```diff
@@ -96,9 +97,13 @@ def can_delete_organisation_records_factory(record, *args, **kwargs):
     """Checks if the logged user can delete records of its organisation.
 
     user must have librarian or system_librarian role.
+    users do not have the permission to delete a monograph holdings record.
     """
     def can(self):
         patron = staffer_is_authenticated()
+        if isinstance(record, Holding) and record.get('holdings_type')\
+                and record.get('holdings_type') == 'monograph':
```

Comment:
this should be done based on issuance. I suggest to have a holding property called hasPatters or isSerial, etc.

Reply:
this has nothing to do with the issuance. this part is to disallow creating of `monograph` holdings by librarians. As we agreed last week, we will remove `later` the `holdings_type` field and replace with `hasPatters or isSerial` or similar.